src/DependencyInjection/Configuration.php

```diff
@@ -28,6 +28,10 @@ private function addConverterSection(ArrayNodeDefinition $rootNode): void
     {
         $rootNode
             ->children()
+                ->scalarNode('converterSuffix')
+                    ->info('consolidated suffix for all converters')
```

Comment:
This message is misleading. It only configures the suffix of the autowiring alias for converter arguments.